### PR TITLE
add documentation for orders query api

### PR DESCRIPTION
**Pull request by P4Awd**

### CODE REVIEW

**Review by rav005 — APPROVED** [review]

File: _orders/querying.md [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+```
+### Step 2: Send a request to list the first 100 orders under your account
+
+Incorporate the access token in the `Authorization` header and send a test request as follows:
+
+```bash
+curl -X POST https://api.pointservices.com/riskinsight-services-ws/resources/v1/orders/query 
+-H "Authorization: Bearer your_access_token_here" 
+-H "Content-Type: application/json" 
+-d ''
```
Comment (lolliopps) [inline _orders/querying.md:55]: write an actual json example instead of leaving it blank

Comment (lolliopps) [reply _orders/querying.md:55]: actually, put a slightly complicated example with 2 items to filter -- that way there's an example at least of how it's written

File: _orders/querying.md [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+    "OwnerUserDisplayName" : "Bloggins, Joe",
+    "ProductDisplayName" : "ADV-AB-BeforeReconfirm",
+    "ProductType" : "ADV",
+    "ReferenceNumber" : "7058774124",
+    "ReportAvailable" : true,
+    "Status" : "complete"
+  }]
+}
+```
+
```
Comment (lolliopps) [inline _orders/querying.md:199]: Right here, before the DSL, I suggest we have 2 new sections.  One is a documentation to describe what the request json will look like.  Specifically, you haven't yet mentioned that limit is 100 when certain conditions are met.  The 2nd is the documentation to describe response.  Specifically count is the number of results returned in this response, and not total number of results.  

I'm actually not sure if we should be returning "Deleted" or not.  Maybe the name should be changed to 'Archived' instead?

**Review by lolliopps — CHANGES_REQUESTED** [review]

**Comment by bhadfield** [comment]
A couple of things that occur to me and don't particularly apply to this PR. Sorry

1. We probably shouldn't have used Dossier. It's debatable because we want the idea that we are not merely transactional but TransactionID is used elsewhere including in the SAMI envelope
2. What happened to the idea of creating a Postman project for users to quickly test while seeing the JSON "live" ?

**Comment by P4Awd** [comment]
> A couple of things that occur to me and don't particularly apply to this PR. Sorry
> 
> 1. We probably shouldn't have used Dossier. It's debatable because we want the idea that we are not merely transactional but TransactionID is used elsewhere including in the SAMI envelope
> 2. What happened to the idea of creating a Postman project for users to quickly test while seeing the JSON "live" ?

1. I've been thinking that dossier id is really a session id. Start a session, do a bunch of work that can span a configurable amount of days (maybe the deletion window), close the session (either automatically or the user can do it). This is an area we need to spend time on to resolve as there is "state" and "orders" that reference what a state looked like at a point in time. If we have a user interface, the interface needs to persist state without ordering which was not baked into our system on day one as we imagined ourselves to be an ordering system. 

2. We could, does this need the enterprise plan?

File: _orders/querying.md (hunk 1 above)
Comment (P4Awd) [reply _orders/querying.md:55]: This is the query for when you don't know what you're querying for.

**Review by lolliopps — APPROVED** [review]

**Pull request merged**